scripts/collect-logs.sh

```diff
@@ -18,6 +18,12 @@ collectCloudProviderJson() {
     if [ -f /etc/kubernetes/azurestackcloud.json ]; then
         jq . /etc/kubernetes/azurestackcloud.json > ${DIR}/azurestackcloud.json
     fi
+    if [ -f /etc/kubernetes/network_interfaces.json ]; then
+        jq . /etc/kubernetes/network_interfaces.json > ${DIR}/network_interfaces.json
+    fi
```

Comment:
This seems to copy files on Linux. What about Windows nodes. We should copy this file even on windows nodes if there is a failure

Reply:
change made